executor/adapter.go

```diff
@@ -390,13 +390,11 @@ func (a *ExecStmt) LogSlowQuery(txnTS uint64, succ bool) {
 	}
 	execDetail := sessVars.StmtCtx.GetExecDetails()
 	if costTime < threshold {
-		if logutil.SlowQueryLogger.IsLevelEnabled(log.DebugLevel) {
```

Comment:
I think this is useful :rofl:

Reply:
It is already guaranteed by https://github.com/pingcap/tidb/pull/9521/files#diff-350127760839dbfd52d23927f7ff2d95R374.